docs/getting-started.md

```diff
@@ -1,372 +1,282 @@
 # Getting Started with Firecracker
 
-## Contents
-
-- [Prerequisites](#prerequisites)
-- [Getting the Firecracker Binary](#getting-the-firecracker-binary)
-- [Running Firecracker](#running-firecracker)
-- [Building From Source](#building-from-source)
-- [Running the Integration Test Suite](#running-the-integration-test-suite)
-- [Appendix A: Setting Up KVM Access](#appendix-a-setting-up-kvm-access)
-- [Appendix B: Setting Up Docker](#appendix-b-setting-up-docker)
-
 ## Prerequisites
 
-If you need an opinionated way of running Firecracker, launch an `i3.metal`
-instance using Ubuntu 18.04 on EC2. Firecracker uses
-[KVM](https://www.linux-kvm.org) and needs read/write access that can be
-granted as shown below:
-
-```console
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
-
-The generic requirements are explained below:
-
-- **Linux**
+You can check if your system meets the requirements by running
+`firecracker/tools/devtool checkenv`.
 
-  Firecracker currently supports physical Linux **x86_64** and **aarch64**
-  hosts. The currently supported kernel versions can be found [here](kernel-policy.md).
+An opinionated way to run Firecracker is to launch an
+[EC2](https://aws.amazon.com/ec2/) `i3.metal` instance with Ubuntu 18.04.
 
-- **KVM**
+Firecracker does not support nested virtualization. This is why `.metal`
+instances are used.
 
-  Please make sure that:
-    1. you have KVM enabled in your Linux kernel, and
-    2. you have read/write access to `/dev/kvm`.
-       If you need help setting up access to `/dev/kvm`, you should check out
-       [Appendix A](#appendix-a-setting-up-kvm-access).
+### Architecture & OS
 
-To check if your system meets the requirements to run Firecracker, clone
-the repository and execute `tools/devtool checkenv`.
+Firecracker supports **x86_64** and **aarch64** Linux, see
+[specific supported kernels](kernel-policy.md).
 
-## Getting the Firecracker Binary
+### KVM
 
-Firecracker is linked statically against
-[musl](https://musl.libc.org/), having no library dependencies. You can
-just download the latest binary from our
-[release page](https://github.com/firecracker-microvm/firecracker/releases),
-and run it on your x86_64 or aarch64 Linux machine.
+Firecracker requires [the KVM Linux kernel module](https://www.linux-kvm.org/).
 
-On the EC2 instance, this binary can be downloaded as:
+The presence of the KVM module can be checked with:
 
-```wrap
-release_url="https://github.com/firecracker-microvm/firecracker/releases"
-latest=$(basename $(curl -fsSLI -o /dev/null -w  %{url_effective} ${release_url}/latest))
-arch=`uname -m`
-curl -L ${release_url}/download/${latest}/firecracker-${latest}-${arch}.tgz \
-| tar -xz
-```
-
-Rename the binary to "firecracker":
-
-```console
-mv release-${latest}-$(uname -m)/firecracker-${latest}-$(uname -m) firecracker
+```bash
+lsmod | grep kvm
```

Comment:
what if kvm is built in the kernel, not compiled as a module (I don't know how common is that)?

Reply:
I'm not aware of this case, do you have any documentation of this you can link?

Reply:
I think, checking for the device file is ok (already mentioned below).
Other ways are hacky (like grepping `/proc/kallsyms`).
Maybe leave the `lsmod` check as is, but also add a note that kvm may be compiled in and still be used, but not visible via `lsmod`?

Reply:
Looks good to me.
```
config KVM
        tristate "Kernel-based Virtual Machine (KVM) support"
```
Tristate usually means enabled, disabled or module.